docs/API_SPECS_DEVICE_MODELS.md

```diff
@@ -1,17 +1,15 @@
 API specifications - Device Models
 ==================================
 
-There are two types of device models:
-1. Stock device models, compiled into the service
-2. Custom device models, created by an application/a user
+## Get list of device models that can be simulated
```

Comment:
could you rebase you branch to the latest master? This would revert many docs changes done recently